packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.html

```diff
@@ -18,19 +19,22 @@
       <button amplify-button variation="primary" fullWidth="true" type="submit">
         {{ signInButtonText }}
       </button>
-      <button
-        amplify-button
-        fontWeight="normal"
-        size="small"
-        variation="link"
-        (click)="authenticator.toResetPassword()"
-      >
-        {{ forgotPasswordText }}
-      </button>
+
       <amplify-error *ngIf="authenticator.error">
         {{ authenticator.error }}
       </amplify-error>
     </fieldset>
-    <amplify-slot name="sign-in-footer"> </amplify-slot>
   </form>
+
+  <amplify-slot name="sign-in-footer">
+    <button
+      amplify-button
+      fontWeight="normal"
+      size="small"
+      variation="link"
+      (click)="authenticator.toResetPassword()"
+    >
+      {{ forgotPasswordText }}
+    </button>
+  </amplify-slot>
```

Comment:
Moving this outside the form means losing the padding & centering:

> <img width="610" alt="Screen Shot 2021-11-16 at 10 26 53 AM" src="https://user-images.githubusercontent.com/15182/142025850-e488360f-6ffc-4476-acb3-da9756c089c9.png">

What's the best way to add this centering back in?

Reply:
Just add styles?